repos/core-setup.proj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <BuildArguments>-ConfigurationGroup=$(Configuration) -PortableBuild=false -SkipTests=true </BuildArguments>
     <BuildArguments Condition="$(Platform.Contains('arm'))">$(BuildArguments) -TargetArchitecture=$(Platform) -DisableCrossgen=true -CrossBuild=true</BuildArguments>
-    <BuildCommand>$(ProjectDirectory)/build$(ShellExtension) $(BuildArguments) -- /p:BuildDebPackage=false</BuildCommand>
+    <BuildCommand>$(ProjectDirectory)/build$(ShellExtension) $(BuildArguments) -- /p:BuildDebPackage=false /p:BuildAllPackages=true</BuildCommand>
```

Comment:
Why did we need to pass BuildAllPackages=true? I wouldn't expect that to be necessary.

Reply:
The CLI build in source-build failed because of 2 missing depencencies: Microsoft.Extensions.DependencyModel and Microsoft.DotNet.PlatformAbstractions.  These are conditionally built based on this property.

Reply:
We intentionally don't build these in servicing, similar to corefx, unless we have changes we want to ship. It has been a while since I've thought about this but it feels like we would have a lot more failures higher up based on not building all the packages. 

I guess for now this should work. For reference the corefx issue I filed a while back is https://github.com/dotnet/source-build/issues/210.